.changeset/spicy-sloths-divide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/unenv-preset": patch
+---
+
+native `node:fs` and `node:os` are no more experimental
```

Comment:
Could you clarify here what the changes do? 🙏 

Something like:
```
enable by default, for compat_dates >= 2025-09-15 the use of the native implementation of `node:fs` and `node:os` since such modules are no longer experimental
```